### PR TITLE
Homework Week 2

### PR DESCRIPTION
**Pull request by kewpiedoll**

### CODE REVIEW

**Comment by kewpiedoll** [comment]
HW 2 (mostly) complete

File: week2/homework/simon_says.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+module SimonSays
+  
+  def echo word
+	 word
+  end
+  
+  def shout word
+  	"#{word.upcase}"
```
Comment (bethanyr) [inline week2/homework/simon_says.rb:8]: in Ruby you can leave off the #{}, in this method you can just call word.upcase

File: week2/homework/simon_says.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  end
+  	
+  def repeat(word)
+    word + ' ' + word
+  end
+
+  def repeat(word, n=2)
```
Comment (bethanyr) [inline week2/homework/simon_says.rb:15]: this code works, but there are simpler ways to code this using Ruby - take a look at the answers and see another way you could code some of the functions. (you'll see why they say Ruby is built for developer happiness)

**Comment by bethanyr** [comment]
Looks good - I reviewed homework for week1 and week 2.

File: week2/homework/simon_says.rb (hunk 1 above)
Comment (ilynne) [inline week2/homework/simon_says.rb:8]: I can't believe I didn't do it that way...

On Thu, Jan 30, 2014 at 8:00 AM, Bethany Rentz notifications@github.comwrote:

> In week2/homework/simon_says.rb:
> 
> > @@ -0,0 +1,56 @@
> > +module SimonSays
> > +
> > -  def echo word
> > -    word
> > -  end
> >   +
> > -  def shout word
> > -   "#{word.upcase}"
> 
> in Ruby you can leave off the #{}, in this method you can just call
> word.upcase
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/UWE-Ruby/RubyWinter2014/pull/36/files#r9310137
> .

## 

Lynne Cooney
425.835.3703

**Pull request closed without merging**